releasenotes/notes/span_aggregator_rlock_default_true-eeab94abb8773480.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    sqlalchemy: sqlalchemy rollbacks could previously cause intermittent deadlocks in some cases.
+    To fix this ``DD_TRACE_SPAN_AGGREGATOR_RLOCK`` was introduced in 1.16.2 with the default as ``False``
+    with the intent to change the default to ``True`` after further testing. Now that further testing has
+    been completed without issue, we are changing the default to ``True``.
```

Comment:
nit: Can we simplify this release note? I don't think we need to mention testing strategy. We should be direct and just call out what was fixed.